libs/types/src/solana/request/mod.rs

```diff
@@ -7,7 +7,7 @@ use candid::{CandidType, Deserialize};
 use serde::Serialize;
 
 /// The parameters for a Solana [`getAccountInfo`](https://solana.com/docs/rpc/http/getaccountinfo) RPC method call.
-#[derive(Debug, Clone, Deserialize, Serialize, CandidType)]
+#[derive(Debug, Clone, Deserialize, Serialize, CandidType, PartialEq)]
```

Comment:
nit: we usually try to follow the following conventions for deriving traits

1. Rust standard trait (often by alphabetical order)
2. Followed by non-standard traits (often by alphabetical order)



```suggestion
#[derive(Clone, Debug, PartialEq, CandidType, Deserialize, Serialize)]
```

Reply:
Ah yes, thanks for pointing that out! Done.